wknml/__init__.py

```diff
@@ -172,24 +171,26 @@ def parse_parameters(nml_parameters):
 
 
 def parse_node(nml_node):
+    rotation = None
+    if nml_node.get("rotX") is not None:
+        rotation = (float(nml_node.get("rotX")),
+        float(nml_node.get("rotY")),
+        float(nml_node.get("rotZ")))
+
     return Node(
         id=int(nml_node.get("id")),
-        radius=float(nml_node.get("radius", default=None)),
+        radius=float(nml_node.get("radius")) if nml_node.get("radius") else None,
```

Comment:
```suggestion
        radius=float(nml_node.get("radius")) if nml_node.get("radius") is not None else None,
```

because a radius of `0.0` would also valuate to `False`, I think. Same for would be for the changes below.

Or, is there something like `nml_node.has_attribute("radius")` method?

Reply:
Sadly I cannot find something like that. `https://docs.python.org/2/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.get`

Reply:
Instead of comparing the file size (which is still working), I am using `filecmp` now.